.gitlab-ci.yml

```
@@ -28,7 +28,7 @@ variables:
   GRADLE_VERSION: "8.14.3" # must match gradle-wrapper.properties
   MAVEN_REPOSITORY_PROXY: "https://depot-read-api-java.us1.ddbuild.io/magicmirror/magicmirror/@current/"
   GRADLE_PLUGIN_PROXY: "https://depot-read-api-java.us1.ddbuild.io/magicmirror/magicmirror/@current/"
-  BUILDER_IMAGE_VERSION_PREFIX: "v25.10-" # use either an empty string (e.g. "") for latest images or a version followed by a hyphen (e.g. "v25.05-")
+  BUILDER_IMAGE_VERSION_PREFIX: "123_merge-" # use either an empty string (e.g. "") for latest images or a version followed by a hyphen (e.g. "v25.05-")
```

Not sure if you requested a review too early, but we'll want this to be:

```suggestion
  BUILDER_IMAGE_VERSION_PREFIX: "v25.11-" # use either an empty string (e.g. "") for latest images or a version followed by a hyphen (e.g. "v25.05-")
```

after https://github.com/DataDog/dd-trace-java-docker-build/pull/123 is merged, and [this workflow](https://github.com/DataDog/dd-trace-java-docker-build/actions/workflows/docker-tag.yml) is run to tag the updated images with the new prefix `v25.11` (2025 Nov) 🙂